app/src/components/dataset/CreateDatasetForm.tsx

```diff
@@ -45,6 +45,25 @@ export function CreateDatasetForm(props: CreateDatasetFormProps) {
         onCompleted: (response) => {
           onDatasetCreated(response["createDataset"]["dataset"]);
         },
+        updater: (store) => {
```

Comment:
finally got this working, not sure what hte problem was before. No longer need to do this https://github.com/Arize-ai/phoenix/issues/4008 which is flakey at best and can start to update connections in teh store directly this way. Can also remove fetch keys etc.